kong/clustering/services/sync/rpc.lua

```diff
@@ -164,6 +167,8 @@ function _M:init_dp(manager)
 
     local lmdb_ver = tonumber(declarative.get_current_hash()) or 0
     if lmdb_ver < version then
+      -- set lastest version to shm
+      kong_shm:set(CLUSTERING_DATA_PLANES_LATEST_VERSION_KEY, version)
```

Comment:
We only run sync in one worker, is it necessary to store it in shared memory?

Reply:
This is not true. @dndx Could you confirm?

Reply:
We run incremental sync inside all workers, it's just that only one worker can sync at the same time.